File: src/main/java/net/hectus/neobb/turn/legacy_game/structure/glass_wall/LTLimeGlassWall.java
package net.hectus.neobb.turn.legacy_game.structure.glass_wall;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.structure.PlacedStructure;
import net.hectus.neobb.turn.default_game.attributes.clazz.NatureClazz;
import net.hectus.neobb.turn.default_game.structure.glass_wall.GlassWallTurn;

public class LTLimeGlassWall extends GlassWallTurn implements NatureClazz {
    public LTLimeGlassWall(NeoPlayer player) { super(player); }
    public LTLimeGlassWall(PlacedStructure data, NeoPlayer player) { super(data, player); }

    @Override
    public String color() {
        return "lime";
    }
}

File: src/main/kotlin/net/hectus/neobb/turn/legacy_game/structure/glass_wall/LTLimeGlassWall.kt
package net.hectus.neobb.turn.legacy_game.structure.glass_wall

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import net.hectus.neobb.structure.PlacedStructure
import net.hectus.neobb.structure.StaticStructure
import net.hectus.neobb.structure.StaticStructures
import net.hectus.neobb.turn.default_game.attribute.clazz.NeutralClazz
import net.hectus.neobb.turn.default_game.structure.glass_wall.GlassWallTurn

class LTLimeGlassWall(data: PlacedStructure?, cord: Cord?, player: NeoPlayer?) : GlassWallTurn(data, cord, player), NeutralClazz {
    override val staticStructure: StaticStructure = StaticStructures.Legacy.GlassWall.LIME
}
